plutus-chain-index/README_alpha-release_Marconi.md

```diff
@@ -0,0 +1,75 @@
+![Marconi-Logo_sm](https://user-images.githubusercontent.com/104791413/178748103-077ed7e0-6071-461f-b9ed-97b3b7bd5b24.png)
```

Comment:
I actually have no idea why this is called Marconi so I have no idea if the logo fits.

Reply:
Since IOG has a tradition of using famous scientists for product names (Ada, Cardano, Basho, Goguen, etc.). Marconi is a famous italian scientist. https://en.wikipedia.org/wiki/Guglielmo_Marconi

I don't think the logo has to fit the person. Similar with Cardano; it's logo doesn't really fit.

Reply:
This is the first idea that we explored with a graphic designer. I basically like it. Taking on logo design can be a big project, and mostly I'm happy we have something we can use. I think it's worth asking the deeper question of what the logo should truly be. But the nuts and bolts of going through a true design process can be very time consuming.

Reply:
I think the logo represents a radio which Marconi invented.